w3-aggregation.md

```diff
@@ -317,12 +294,14 @@ If offered aggregate is invalid, details on failing commitmentProofs are also re
 }
 ```
 
-### Schema
+## Schema
+
+### Base types
 
 ```ipldsch
 type Aggregate union {
   | Link "queued"
-  | Link "status"
+  | Link "accepted"
   | Link "rejected"
 } representation keyed
```

Comment:
Is this a response object for the `aggregate/get` ? I can't see it referenced so it's not very clear.  Maybe adding a comment not top telling what is it would help.

It is also unclear what all the `Link`s supposed to point at. Do you mind using `&Accepted` type links and define schemas for individual cases as well ?